### PR TITLE
Improve title in void-dom-elements-no-children.md

### PR DESCRIPTION
**Pull request by omerzach**
Add the name of the rule in parentheses at the end of Markdown title like in other documentation files.

### CODE REVIEW

File: docs/rules/void-dom-elements-no-children.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children
+# Prevent void DOM elements (e.g. `<img />`, `<br />`) from receiving children (void-dom-elements-no-children)
```
Comment (ljharb) [inline docs/rules/void-dom-elements-no-children.md]: Per #1071, this should probably have a "react/" prefix.

**Comment by omerzach** [comment]
Ah I was hoping I could squash the commits from within the GitHub UI but I guess not

**Comment by omerzach** [comment]
I'm bad at GitHub so I'm going to close this and open a new one. Sorry.

**Comment by ljharb** [comment]
Yeah you need to use the command line - in the future, please have a collab do it for you instead of opening up a duplicate PR

**Pull request closed without merging**